### ISSUE TITLE
Design project view

### ISSUE THREAD

**Issue by NealHumphrey** [codefordc/housing-insights#399]
We need lots more information in the project view (formerly called building view) Before implementing this, we need to design what this view will include. 

Note, some basic additions are being added in #398

**Comment by ableo57** [comment]
@NealHumphrey please find an updated draft here: https://app.moqups.com/neal@nhumphrey.com/zeZOXaXpNx/edit/page/ad3aea595

**Comment by ableo57** [comment]
Notes from my conversation with Neal:

![image](https://user-images.githubusercontent.com/24680097/28395320-25237d00-6cc1-11e7-8862-2d7a19f1f72f.png)

**Comment by NealHumphrey** [comment]
@ableo57 Just want to check in on this - how's design coming? Let me know if you need anything, hoping to turn this into an action plan at tonight's session!

**Comment by ableo57** [comment]
@NealHumphrey 

There are a few activities going on:

1. @txomashernandez is working to send out a survey to refine which metrics users would find helpful in the project view. Essentially, we're presenting an exhaustive list of variables and having respondents rate their importance on a likert scale

2. Dashboard graphics -- I've mocked up a few different options for both contextual and time-evolved graphics. I'm working to think through others as well.

3. Putting it all together/grouping: Thomas mentioned Jaime would be the best to put together the final organization of the mockup. Hoping to do that tonight, but if Jaime isn't able to come, I'll work at it and take care of it by the end of tomorrow.

Let me know if we've got developers chomping at the bit.
